packages/functional-tests/tests/react-conversion/oauthPromptNone.spec.ts

```diff
@@ -220,6 +220,36 @@ test.describe('severity-1 #smoke', () => {
       ).toBeVisible();
     });
   });
+
+  test('redirects if return_on_error=true and not signed in', async ({
+    page,
+    target,
+    pages: { relier, settings, signin },
+    testAccountTracker,
+  }) => {
+    {
+      const { email } = await signInAccount(
+        target,
+        page,
+        settings,
+        signin,
+        testAccountTracker
+      );
+
+      await settings.signOut();
+
+      const query = new URLSearchParams({
+        login_hint: email,
+        return_on_error: 'true',
+      });
+      await page.goto(`${target.relierUrl}/?${query.toString()}`);
+      await relier.signInPromptNone();
+
+      // RP handled it by taking the user back to sign in
```

Comment:
Would the test be more robust with a verification that that the redirect to RP occurs? From the steps here it looks like the redirect to sign in is handled by the index page and not the relying party.

Reply:
Is there an example of how we verify there was an intermediate state?  I guess look at the urls in the history?  I'll give it a try.

Reply:
Added a `page.waitForResponse`.  @vpomerleau thanks for the call out.